deploy/helm/gateway-collector-config.yaml

```diff
@@ -0,0 +1,185 @@
+        action: update
+        new_name: k8s.$${1}
+
+  resource:
+    attributes:
+      # Collector and Manifest version
+      - key: sw.k8s.agent.manifest.version
+        value: ${MANIFEST_VERSION}
+        action: insert
+
+      - key: sw.k8s.agent.app.version
+        value: ${APP_VERSION}
+        action: insert
+
+      # Cluster
+      - key: sw.k8s.cluster.uid
+        value: ${CLUSTER_UID}
+        action: insert
+
+      - key: k8s.cluster.name
+        value: ${CLUSTER_NAME}
+        action: upsert
+
+  # Those attributes are defined by SWO K8s Collector even if they are send as attributes in OTLP message
+  attributes/clean-attributes-otlp-metrics:
+    actions:
+      - key: k8s.cluster.name
+        action: delete
+
+  transform/scope:
+    metric_statements:
+      - statements:
+        - set(scope.name, "")
+        - set(scope.version, "")
+    log_statements:
+      - statements:
+        - set(scope.name, "")
+        - set(scope.version, "")
+    trace_statements:
+      - statements:
+        - set(scope.name, "")
+        - set(scope.version, "")
+
+  filter/otlp-metrics:
+    metrics:
+      metric:
+        # metrics from EBPF reducer that are not needed
+        - 'IsMatch(name, "^ebpf_net.*$")'
+
+  filter/histograms:
+    metrics:
+      metric:
+        # Exclude all histograms except the ones listed below
+        # excluded histograms are:
+        # - all histograms required for system dashboards (apiserver, coredns, controller_manager)
+        # - all histograms from beyla
+        - 'type == METRIC_DATA_TYPE_HISTOGRAM and not(
+            name == "k8s.apiserver_request_duration_seconds" or 
+            name == "k8s.workqueue_queue_duration_seconds" or 
+            name == "k8s.coredns_dns_request_duration_seconds" or 
+            name == "k8s.coredns_dns_request_size_bytes" or 
+            name == "k8s.coredns_dns_response_size_bytes" or 
+            name == "k8s.http.client.request.duration" or
+            name == "k8s.http.client.request.body.size" or
+            name == "k8s.http.server.request.duration" or 
+            name == "k8s.http.server.request.body.size" or
+            name == "k8s.rpc.client.duration" or
+            name == "k8s.rpc.server.duration" or
+            name == "k8s.sql.client.duration" or
+            name == "k8s.redis.client.duration" or
+            name == "k8s.messaging.publish.duration" or
+            name == "k8s.messaging.process.duration"
+          )'
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:{{ .Values.otel.gateway.otlp_endpoint.port }}
+      http:
+        endpoint: 0.0.0.0:{{ .Values.otel.gateway.otlp_endpoint.http_port }}
+
+service:
+  extensions:
+{{- if .Values.otel.gateway.sending_queue.offload_to_disk }}
+    - file_storage/sending_queue
+{{- end }}
+    - health_check
+  pipelines:
+    metrics:
+      exporters:
+        - otlp
+      processors:
+        - memory_limiter
+        - filter/otlp-metrics
```

Comment:
What about the check for `.Values.ebpfNetworkMonitoring.reducer.telemetry.metrics.enabled` in the original pipeline? Is it not needed anymore?

Reply:
Ok adding.